data/templates/esx-ks

```diff
@@ -209,7 +209,23 @@ cp /var/log/esxi_install.log "/vmfs/volumes/datastore1/firstboot-esxi_install.lo
 #
 # NOTE: this script will execute right away as a result of writing it to local.sh
 # along with executing on every subsequent boot
-wget http://<%=server%>:<%=port%>/api/current/templates/<%=rackhdCallbackScript%>?nodeId=<%=nodeId%> -O /etc/rc.local.d/local.sh
+#
+# Try to download call back script 60 times 1 second
+# sleep in between to allow link to be up after DHCP
+for retry in $(seq 1 60);
+do
+    wget http://<%=server%>:<%=port%>/api/current/templates/<%=rackhdCallbackScript%>?nodeId=<%=nodeId%> -O /etc/rc.local.d/local.sh
+    if [ $? -eq 0 ]; then
+        logger -p user.info "RackHD's call back script was downloaded successfully after $retry attempt(s)."
+        break
+    else
+        logger -p user.info "Failed to download RackHD's call back script after $retry attempt(s)."
+        sleep 1
+        if [ $retry -eq 60 ]; then
```

Comment:
You should be able to just check it once after the loop is finished instead of every iteration.